src/components/elements/input/ClientImageUploadDialog.tsx

```diff
@@ -163,14 +164,12 @@ const ClientImageUploadDialog: React.FC<ClientImageUploadDialogProps> = ({
               <Grid item xs={12}>
                 <Uploader
                   id='clientImageUploader'
-                  onUpload={(upload, file) => {
-                    setNewBlobId(upload.signedBlobId);
-                    setNewPhotoSrc(URL.createObjectURL(file));
-                    setErrors(emptyErrorState);
-                  }}
-                  onClear={() => {
-                    setNewBlobId(undefined);
-                    setNewPhotoSrc(undefined);
+                  onUpload={(
+                    upload: DirectUpload | undefined,
+                    file: File | undefined
+                  ) => {
```

Comment:
```suggestion
                  onUpload={(upload, file) => {
```
The explicit types shouldn't be needed here, I don't think